processing/src/main/java/io/druid/query/groupby/epinephelinae/GroupByQueryEngineV2.java

```diff
@@ -113,6 +114,13 @@ private GroupByQueryEngineV2()
         null
     );
 
+    final boolean allSingleValueDims = query.getDimensions().stream()
```

Comment:
This kind of chaining indentation is unconvenient IMO. It forces you to break lines because the indentation is too deep. I would break like
```java
final boolean allSingleValueDims = query
    .getDimensions()
    .stream()
    ...
```